tests/read/coff.rs

```diff
@@ -21,3 +23,110 @@ fn coff_extended_relocations() {
     let relocations = code_section.relocations().collect::<Vec<_>>();
     assert_eq!(relocations.len(), 65536);
 }
+
+#[test]
+fn coff_imports() {
```

Comment:
Instead of having this test, put the objdump output under crates/examples/testfiles.

(The `coff_extended_relocations` test is done here because the output would be quite large.)